BrainPortal/cbrain_plugins/cbrain-plugins-base/cbrain_task/boutiques_task/views/_show_params.html.erb

```diff
@@ -24,12 +24,24 @@
 
 <%-
   # Parameter groups
-  @descriptor = @task.descriptor_for_show_params
-  inputs      = @descriptor.inputs
-  file_inputs = @descriptor.file_inputs
-  params      = inputs.select { |i| i.type != 'File' }
-  outputs     = @descriptor.output_files
+  @descriptor     = @task.descriptor_for_show_params
+  inputs          = @descriptor.inputs
+  file_inputs     = @descriptor.file_inputs
+  file_inputs_ids = file_inputs.map(&:id)
```

Comment:
Why is this defined here, and not used? At line 37 you do again the same `file_inputs.map(&:id)` ?